packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -1214,10 +1214,10 @@ export default class TradeStore extends BaseStore {
     }
 
     async getFirstOpenMarket(markets_to_search) {
```

Comment:
use `@action.bound` annotation. That should fix undefined `this`

Reply:
Thanks, this is indeed a better solution